uswds/uswds-site
# USWDS-Site - Header: Add variants table to guidance page

# Summary

**Added header variants table to guidance page.** This change also removes empty "**Using the header**" section.


## Related issue

Closes #1976 

## Preview link

Preview link:
[Header variants →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-header-variant-guidance/components/header/#header-variants)
[Header changelog →](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-header-variant-guidance/components/header/#latest-updates)

## Problem statement

The Header guidance page was missing a variants table that can be found in other component guidance pages. There were also empty implementation sections due to each component containing variants in its frontmatter, which were not displayed in the sections.

## Solution

Move the `variants` out of the individual header variants front matter and include it in the base header guidance markdown page.

## Testing and review

1. Visit [Header guidance page](https://federalist-ead78f8d-8948-417c-a957-c21ec5617a57.sites.pages.cloud.gov/preview/uswds/uswds-site/cm-header-variant-guidance/components/header/)
2. View variants table and check for missing variants
3. Check each variant section and check for empty implementation sections _(screenshots below)_

### Screenshots

### Empty implementation section
_There's an empty **Using the...** section for each variant_
![image](https://github.com/uswds/uswds-site/assets/25211650/62518b4b-2ecb-4b68-af35-20868462a65b)
[As seen on live site →](https://designsystem.digital.gov/components/header/#:~:text=of%20a%20megamenu.-,Using%20the%20basic%20header,-Package)

### New variants table
![image](https://github.com/uswds/uswds-site/assets/25211650/0a6c2f9c-d379-4f2b-8de7-7ad36e537723)





Before opening this PR, make sure you’ve done whichever of these applies to you:
